pkg/streamingpromql/operators/functions/rate_increase.go

```diff
@@ -289,3 +280,94 @@ func rateSeriesValidator() RangeVectorSeriesValidationFunction {
 		lastCheckedMetricName = metricName
 	}
 }
+
+func delta() RangeVectorStepFunction {
+	return func(step *types.RangeVectorStepData, rangeSeconds float64, emitAnnotation types.EmitAnnotationFunc) (float64, bool, *histogram.FloatHistogram, error) {
```

Comment:
I think this can be simplified to this - there's no need for the wrapper function given we don't need different functionality like we do for `rate` and `increase`:

```suggestion
func delta(step *types.RangeVectorStepData, rangeSeconds float64, emitAnnotation types.EmitAnnotationFunc) (float64, bool, *histogram.FloatHistogram, error) {
```